### PR TITLE
Add setProxy method

### PR DESCRIPTION
**Pull request by wesleymilan**
### Proposed changes in this pull request

Just add the PhantomJS method setProxy

#### Checklist
* [ ] New tests have been added
* [ ] `npm test` passes successfully
* [ ] Documentation has been updated

@amir20 to review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 node_modules
-lib/
```
Comment (amir20) [inline .gitignore]: Uh...can you fix this? The lib dir should be ignored

**Comment by amir20** [comment]
Where are the tests and docs?

**Comment by elias-winberg** [comment]
It looks good, but you shouldn't commit the compiled code (i.e., don't add the 'lib' directory). You should probably add a test too.

**Comment by elias-winberg** [comment]
Any idea on how we could test this?

**Comment by amir20** [comment]
Let me think about that. I couldn't find anything about set proxy documentation so I am not sure how to call it. At the very least, we should make sure that no errors are raised when it is called. I was also thinking maybe setting bad proxy to see if it fails.

**Comment by amir20** [comment]
@wesleymilan any update?

**Comment by elias-winberg** [comment]
Uh, it looks like you have removed `#setProxy` now? If you want to add other methods you should probably do that in another pull request.

**Comment by amir20** [comment]
Hey @wesleymilan, unfortunately this pull request is not up to this repo's standards. Please review the [guidelines](https://github.com/amir20/phantomjs-node/blob/master/.github/CONTRIBUTING.md) for creating pull requests. Specifically, there must be new tests. Adding new functionality without tests will not be merged. There are also build errors in Travis and you have reversed the .gitignore file. Due to those reasons, I will close this pull request. Please create a new pull request with all these fixes.

**Pull request closed without merging**